### DISCUSSION TITLE
示例中的ER图如何实现子节点中鼠标监控事件

### DISCUSSION THREAD

**Discussion by HuiTurn** [antvis/X6#2266] [Q&A]
根据查看文档说明，只要针对边节点的鼠标事件监控，我想实现当鼠标悬浮在字段上方时显示该字段的提示详细，
![图片](https://user-images.githubusercontent.com/28799892/175815543-39d68c1e-6c7a-4f62-bb6c-f2c9ed8bc3d4.png)

![图片](https://user-images.githubusercontent.com/28799892/175815835-c19940a1-d5de-49d7-81eb-e383b5e067d0.png)

**Comment by tounks** [comment]
同样有此问题哦
